frameworks/Python/tornado/commons.py

```diff
@@ -1,4 +1,4 @@
-import json
+import ujson as json
```

Comment:
Why ujson? It seems ujson is dead. Lastest version v1.3.5 was released 3 years ago.

Reply:
> frameworks/Python/aiohttp/requirements.txt:ujson==1.35
> frameworks/Python/blacksheep/requirements.txt:ujson==1.35
> frameworks/Python/bottle/requirements.txt:ujson==1.35
> frameworks/Python/django/requirements.txt:ujson==1.35
> frameworks/Python/django/requirements_py3.txt:ujson==1.35
> frameworks/Python/fastapi/requirements.txt:ujson==1.35
> frameworks/Python/japronto/requirements.txt:ujson==1.35
> frameworks/Python/responder/requirements.txt:ujson==1.35
> frameworks/Python/starlette/requirements.txt:ujson==1.35
> frameworks/Python/uvicorn/requirements.txt:ujson==1.35
> frameworks/Python/uwsgi/requirements.txt:ujson==1.35
> frameworks/Python/vibora/requirements.txt:ujson==1.35
> frameworks/Python/wsgi/requirements.txt:ujson==1.35
It's used in multiple places.

Reply:
Looking at usjon's opening issues and current status I will not consider using usjon in production. Just my two cents.